components/backup/src/endpoint.rs

```diff
@@ -1107,7 +1107,9 @@ pub mod tests {
                     let mut endpoint = endpoint.lock().unwrap();
                     endpoint.run(task);
                 }
-                endpoint.lock().unwrap().on_timeout(&mut backup_timer, ());
+                if let Ok(mut e) = endpoint.lock() {
```

Comment:
PTAL @NingLin-P

Reply:
I think use `.unwrap()` is ok for `Mutex::lock`, if an `Err` is return means another thread is panicked, we can also panic here.